steps/image_steps.py

```diff
@@ -3,12 +3,7 @@
 
 from behave import then
 
-
-try:
-    DOCKER_CLIENT = docker.Client(version="1.22")
-except:
-    DOCKER_CLIENT = docker.APIClient(version="1.22")
-
+DOCKER_CLIENT = docker.APIClient()
```

Comment:
I'm wondering if we should explicitly specify the version e.g. akin to https://github.com/cekit/cekit/pull/902 ?

Reply:
Can we change this to match the latest CEKit to specify a version and make it match CEKit. Ideally we'd use the constant form CEKit but until the next release I don't think we can do that. Perhaps hard code and log a separate issue to change it on the next release?

Reply:
Hi! Sure of course. How's 3dcbfe5 ?